crates/bevy_render/src/view/window/mod.rs

```diff
@@ -42,7 +42,7 @@ impl Plugin for WindowRenderPlugin {
                 .init_resource::<WindowSurfaces>()
                 .init_non_send_resource::<NonSendMarker>()
                 .add_systems(ExtractSchedule, extract_windows)
-                .configure_set(Render, WindowSystem::Prepare.in_set(RenderSet::Prepare))
+                .configure_set(Render, WindowSystem::Prepare.in_set(RenderSet::ManageViews))
```

Comment:
i removed all the other stage wrappers but missed this one, could we please remove `WindowSystem::Prepare` as well (and make prepare_view_targets `.after(prepare_windows)` directly)?